sale_sourced_by_line/tests/test_sale_sourced_by_line.py

```diff
@@ -8,7 +8,7 @@
 
 class TestSaleSourcedByLine(TransactionCase):
```

Comment:
FYI, a better test base class is : https://github.com/odoo/odoo/blob/140e58c5a68db674fc1d240147e4d669b29f2cad/odoo/addons/base/tests/common.py#L30

in order to include performances enhancement (DISABLED_MAIL_CONTEXT)

Reply:
thanks, done